pkg/storage/legacysql/context.go

```diff
@@ -0,0 +1,21 @@
+package legacysql
```

Comment:
^^^ this causes circular reference... so needs a new package

BUT I don't like this approach anyway